compiler/damlc/tests/src/DA/Test/ShakeIdeClient.hs

```diff
@@ -864,20 +870,87 @@ visualDamlTests = Tasty.testGroup "Visual Tests"
                 , "  where"
                 , "    signatory owner"
                 , "    controller owner can"
-                , "      nonconsuming ReducedCoin : ContractId Coin"
+                , "      nonconsuming ReducedCoin : ()"
                 , "        with otherCoin : ContractId Coin"
                 , "        do "
-                , "        cn <- fetch otherCoin "
-                , "        create this with amount = cn.amount - 10"
+                , "        cn <- fetch otherCoin"
+                , "        archive otherCoin"
```

Comment:
Do we need the `archive` here if we just want to ensure that `fetch` does not produce an edge in the graph?

Reply:
Nope,  got rid of it